compass/landice/tests/ismip6_forcing/ocean_thermal/__init__.py

```diff
@@ -0,0 +1,39 @@
+from compass.testcase import TestCase
+from compass.landice.tests.ismip6_forcing.ocean_thermal.\
+    process_thermal_forcing import ProcessThermalForcing
+from compass.landice.tests.ismip6_forcing.configure import configure as \
+    configure_testgroup
+
+
+class OceanThermal(TestCase):
+    """
+    A test case for processing the ISMIP6 ocean thermal forcing data.
+    The test case builds a mapping file for interpolation between the
+    ISMIP6 8km polarstereo grid and MALI mesh, regrids the forcing data
+    and renames the ISMIP6 variables to corresponding MALI variables.
+    """
```

Comment:
```suggestion
    ISMIP6 8km polarstereo grid and MALI mesh, regrids the forcing data
    and renames the ISMIP6 variables to corresponding MALI variables.

    Attributes
    ----------
    process_obs : bool
        Whether we are processing observations rather than CMIP model data
    """
```